### PR TITLE
add updated overview page  [beta]

### PR DESCRIPTION
**Pull request by mirnawong1**
this pr adds updated images/gifs related to the new explorer overview page, as well as new info to the 'latest updates' section too. 

[  ] PM review required
[  ] Docs review required

### CODE REVIEW

File: website/docs/docs/collaborate/explore-projects.md [hunk 1]
```diff
@@ -15,7 +15,17 @@ With dbt Explorer, you can view your project's [resources](/docs/build/projects)
...
 
-<Lightbox src="/img/docs/dbt-cloud/explore-nav.jpg" width="95%" title="Access dbt Explorer from dbt Cloud by clicking Explore in the navigation."/>
+## Overview page
+
+Navigate the dbt Explorer overview page to access your project's resources and metadata. The page includes the following sections:
+
+- **Search bar** &mdash; [Search](#search-resources) for resources in your project by keyword. You can also use filters to refine your search results.
+- **Sidebar** &mdash; Use the left sidebar to access model [performance](/docs/collaborate/model-performance), [project recommendations](/docs/collaborate/project-recommendations) in the **Project details** section. Browse your project's [resources, file tree, and database](#browse-with-the-sidebar) in the lower section of the sidebar.
+- **Lineage graph** &mdash; Explore your project's or account's [lineage graph](#project-lineage) to visualize the relationships between resources.
+- **Latest updates** &mdash; View the latest changes or issues related to your project's resources, including the most recent job runs, changed properties, lineage, issues, and more.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/explore-projects.md]: @cafzal do we need to add anything about when the 'latest updates' are updated? and is the content right?

File: website/docs/docs/dbt-versions/release-notes.md [hunk 2]
```diff
@@ -19,6 +19,8 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 [^*] The official release date for this new format of release notes is May 15th, 2024. Historical release notes for prior dates may not reflect all available features released earlier this year or their tenancy availability.
 
 ## July 2024
+
+- **Enhancement**: Introducing a revamped overview page for dbt Explorer, which includes a new design and layout for the Explorer homepage. The new layout provides a more intuitive experience for users to navigate their dbt projects, as well as a new **Latest updates** section to view the latest changes or issues related to project resources. To learn more, refer to [Overview page](/docs/collaborate/explore-projects#overview-page).
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes.md]: release note entry here

File: website/docs/docs/collaborate/explore-projects.md (hunk 1 above)
Comment (matthewshaver) [inline website/docs/docs/collaborate/explore-projects.md]: Small suggestion so folks aren't left wondering at the end of the sentence since we hit the big blocks there
```suggestion
- **Latest updates** &mdash; View the latest changes or issues related to your project's resources, including the most recent job runs, changed properties, lineage, and issues.
```

File: website/static/img/docs/collaborate/dbt-explorer/explorer-model-performance.gif
Comment (matthewshaver) [inline website/static/img/docs/collaborate/dbt-explorer/explorer-model-performance.gif:1]: This GIF might be a little long and sort of jumps around in a few spots, making it hard to follow. GIFs should be smooth flowing and short (1-3 seconds) since users would have to sit through it again to see what they missed. Would maybe better split up or re-created as a video walkthrough, not a blocker for this PR, could be a follow up.

**Review by matthewshaver — APPROVED** [review]

File: website/static/img/docs/collaborate/dbt-explorer/explorer-model-performance.gif
Comment (mirnawong1) [reply website/static/img/docs/collaborate/dbt-explorer/explorer-model-performance.gif:1]: thanks Matt! will address this in the next product cycle wave since it's beta
